src/main/java/Event.java

```diff
@@ -0,0 +1,28 @@
+public class Event extends Task {
+
+    protected String task;
+    protected String from;
+    protected String to;
+
+    public Event(String description) {
+        super(description);
+        this.task = this.description.substring(
+                "event".length(), 
```

Comment:
it's great that you wrapped lines with 8 spaces indentation

Reply:
Thanks!